quantum/quantum.c

```diff
@@ -89,7 +89,7 @@ static void do_code16(uint16_t code, void (*f)(uint8_t)) {
 
     uint8_t mods_to_send = 0;
 
-    if (code & QK_RMODS_MIN) { // Right mod flag is set
+    if (code & QK_RMODS_MIN) {  // Right mod flag is set
```

Comment:
```suggestion
    if (code & QK_RMODS_MIN) { // Right mod flag is set
```

Reply:
I can fix it, but this was a result of running `qmk cformat quantum/quantum.c`....  So it may be ran on this again and apply this after merging.

Reply:
Ah, the Google style has `SpacesBeforeTrailingComments: 2`. Personally I feel that should be 1... 😄

Reply:
Well, we *could* change that (in the next breaking changes :D )

But yeah, I've added the reformatting into my workflow, so this just happened to get hit.

Reply:
> Well, we _could_ change that (in the next breaking changes :D )

Which is set to close to new PRs in [two days](https://docs.qmk.fm/#/breaking_changes?id=when-is-the-next-breaking-change).